packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback } from 'react';
+import { useBooleanState } from 'fxa-react/lib/hooks';
+import UnitRow from '../UnitRow';
+import Modal from '../Modal';
+import AlertBar from '../AlertBar';
+
+type UnitRowSecondaryEmailProps = {
+  primaryEmail: string;
+};
+
+export const UnitRowSecondaryEmail = ({ primaryEmail }: UnitRowSecondaryEmailProps) => {
+  const [alertBarRevealed, revealAlertBar, hideAlertBar] = useBooleanState();
+  const [modalRevealed, revealModal, hideModal] = useBooleanState();
+
+  const modalHeaderId = 'modal-header-verify-email';
+  const modalDescId = 'modal-desc-verify-email';
+
+  const onSecondaryEmailConfirm = useCallback(() => {
+    console.log('confirmed - resend verification code');
```

Comment:
from debugging?

Reply:
Yeah, we've had this in there. This will actually be done/taken out in #4903.